checker/localizations_src/en/messages.yaml

```
@@ -166,3 +166,11 @@ response-property-pattern-added: the %s response's property pattern %s was added
 response-property-pattern-removed: the %s response's property pattern %s was removed for the status %s
 response-property-default-value-changed: the %s response's property default value changed from %s to %s for the status %s
 response-body-default-value-changed: the response body %s default value changed from %s to %s for the status %s
+request-body-any-of-added: added '%s' to the request body 'anyOf' list
```

not sure what the standard practice is on the copy but `%q` adds quotes 
```suggestion
request-body-any-of-added: added %q to the request body 'anyOf' list
```

---

%q will add double-quotes (`a double-quoted string safely escaped with Go syntax`) which won't be consistent with the rest of the messages oasdiff already has

---

on this, one thing you can do is to keep %s and print with `ColorizedValue` which is something I've seen across the repo

---

good idea, done